py-polars/Makefile

```diff
@@ -91,53 +93,55 @@ clippy:  ## Run clippy
 pre-commit: fmt clippy  ## Run all code quality checks
 
 .PHONY: test
-test: develop # develop  ## Run fast unittests
-	$(VENV_ACT_BIN)pytest tests/unit/
+test: $(VENV)  ## Run fast unittests
```

Comment:
`make test` should run out of the box. Currently it says 'no module named polars' because it does not call `maturin develop`.